docs/api/setup-server/listen.mdx

```diff
@@ -16,12 +16,15 @@ Specifies how to handle a request that is not listed in the request handlers.
 
 #### Pre-defined behaviors
 
+With these options, the unhandled request gets performed and a message is logged. If you want to prevent the request altogether, you can use a custom callback.
```

Comment:
That is not true as the "error" option does not perform the request. I think we should remove this paragraph to prevent confusion.